src/rustdoc.md

```diff
@@ -34,6 +34,54 @@ does is call the `main()` that's in this crate's `lib.rs`, though.)
 
 [bin]: https://github.com/rust-lang/rust/tree/master/src/tools/rustdoc
 
+## Fast builds
+
+Preconfigure your `/config.toml`:
+
+```toml
+# =============================================================================
+# Global Settings
+# =============================================================================
+
+# Use different pre-set defaults than the global defaults.
+#
+# See `src/bootstrap/defaults` for more information.
+# Note that this has no default value (x.py uses the defaults in `config.toml.example`).
+#profile = <none>
+#
+# make sure that you do not have a profile set
+
+# =============================================================================
+# Tweaking how LLVM is compiled
+# =============================================================================
+[llvm]
+
+# Whether to use Rust CI built LLVM instead of locally building it.
+#
+# Unless you're developing for a target where Rust CI doesn't build a compiler
+# toolchain or changing LLVM locally, you probably want to set this to true.
+#
+# This is false by default so that distributions don't unexpectedly download
+# LLVM from the internet.
+#
+# All tier 1 targets are currently supported; set this to `"if-supported"` if
+# you are not sure whether you're on a tier 1 target.
+#
+# We also currently only support this when building LLVM for the build triple.
+#
+# Note that many of the LLVM options are not currently supported for
+# downloading. Currently only the "assertions" option can be toggled.
+download-ci-llvm = true
+```
+
+Now build ´rustdoc´ from stage 2 and build the documentation of the standard library:
+
+```bash
+x.py doc std --stage 2 --open
```

Comment:
Normally, `stage 1` is supposed to be working. Maybe open an issue on the rust repository about it?

Reply:
I would rather just recommend  --stage 1.

Reply:
I did rebuild it with the current `master` something fixed the build process there.
Now `stage 1` works too, but it is ~38min on an 8 core CPU.

Reply:
Stage 1 should never be slower than stage 2. I saw below that stage2 took you about 23 minutes - did you maybe have rustdoc already compiled or something?

Reply:
I enabled `[llvm] download-ci-llvm = true` in `/config.toml` so I did not compile LLVM if I understand correctly

Reply:
Ok, cool. Let's change this to --stage 1 then (or just omit the stage, the default is correct).